dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -575,6 +576,8 @@ public void setWorkflowVersionLimit(int workflowVersionLimit) {
 
         private boolean enableLaunchWithFireCloud;
 
+        private Optional<String> tagManagerId;
```

Comment:
Not a fan about making this optional contrary to the pattern above with client  and secret ids.  
I think I'd rather a developer have to restart the webservice occasionally with a silly value than forget to include the value in staging/prod and have to deploy.